applications/conversations/models/class.conversationsmodel.php

```diff
@@ -13,6 +13,13 @@
  */
 abstract class ConversationsModel extends Gdn_Model {
 
+    use \Vanilla\FloodControlTrait;
+
+    /**
+     * @var \Vanilla\CacheInterface Object used to store the FloodControl data.
+     */
+    protected $floodControlStorageObject;
```

Comment:
`$floodControlTracker` or `$floodgate` or even `$floodController` would be nicer. "StorageObject" makes it sound like I woke up in a Java shop.